packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts

```diff
@@ -335,8 +335,8 @@ export const setTeamDescription: Query<
 `;
 
 export const unmakeSandboxesTemplate: Query<
```

Comment:
Should we add a deprecation flag for these functions? Or is it not the case that they're deprecated?

Reply:
https://www.notion.so/codesandbox-app/Tech-debt-accounting-c4e9e7420721494faa22b47cbb35cc10?p=a9e05cf80fb147a8b71d72a3a165a0ca&pm=s